### PR TITLE
[CWS] use the correct /proc during snapshot

### PR DESCRIPTION
**Pull request by Gui774ume**
### What does this PR do?

Fix a tiny bug in the snapshot phase for bind events in activity dumps.

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by spikat — APPROVED** [review]

File: pkg/security/probe/activity_dump.go [hunk 1]
```diff
@@ -966,10 +966,11 @@ func (pan *ProcessActivityNode) snapshotBoundSockets(p *process.Process, ad *Act
 		return err
 	}
 
+	// sockets have the following pattern "socket:[inode]"
 	var sockets []uint64
 	for _, fd := range FDs {
-		if socketRe.MatchString(fd.Path) {
```
Comment (paulcacheux) [inline pkg/security/probe/activity_dump.go:971]: I think you can remove the regexp definition now

File: pkg/security/probe/activity_dump.go [hunk 2]
```diff
@@ -983,9 +984,8 @@ func (pan *ProcessActivityNode) snapshotBoundSockets(p *process.Process, ad *Act
 		return nil
 	}
 
-	// init procfs to parse /proc/net/tcp,tcp6,udp,udp6 files, looking for the grabbed
-	// process socket inodes
-	proc, _ := procfs.NewFS(util.HostProc())
+	// use /proc/[pid]/net/tcp,tcp6,udp,udp6 to extract the ports opened by the current process
+	proc, _ := procfs.NewFS(filepath.Join(util.HostProc(), fmt.Sprintf("%d", p.Pid)))
```
Comment (paulcacheux) [inline pkg/security/probe/activity_dump.go:988]:
```suggestion
	proc, _ := procfs.NewFS(filepath.Join(util.HostProc(fmt.Sprintf("%d", p.Pid))))
```
```suggestion
	proc, _ := procfs.NewFS(filepath.Join(util.HostProc(), fmt.Sprintf("%d", p.Pid)))
```

**Review by paulcacheux — APPROVED** [review]

File: pkg/security/probe/activity_dump.go (hunk 2 above)
Comment (spikat) [reply pkg/security/probe/activity_dump.go]: updated, thanks

File: pkg/security/probe/activity_dump.go (hunk 1 above)
Comment (spikat) [reply pkg/security/probe/activity_dump.go:971]: fixed, thanks

**Pull request merged**